Sources/EpoxyBars/BarModel/AnyBarModel.swift

```diff
@@ -60,3 +61,51 @@ extension AnyBarModel: Diffable {
     return model.isDiffableItemEqual(to: otherDiffableItem)
   }
 }
+
+// MARK: CallbackContextEpoxyModeled
+
+extension AnyBarModel: CallbackContextEpoxyModeled {
+  public typealias CallbackContext = Void
```

Comment:
On the `AnyItemModel` there's more info available in the `CallbackContext`, would we be able to include some of those here as well? https://github.com/airbnb/epoxy-ios/blob/c82bda711ac0569bc4dcc9118a4c2f89ad6ed24f/Sources/EpoxyCollectionView/Models/ItemModel/AnyItemModel.swift#L150-L180

Reply:
At least for `willDisplay` this should be doable considering that it's called directly next to the underlying `willDisplay` call.